### DISCUSSION TITLE
rewriteOut not applied when specified in client config block

### DISCUSSION THREAD

**Discussion by joeselway** [radsecproxy/radsecproxy#150] [Q&A]
I have a simple rewrite rule which was not being applied when included as rewriteOut within the client block.

Moving the same rewrite to the server block, it is applied successfully.

If this is intended behaviour (.e.g rewriteIn applies only for client while rewriteOut applies only for server?), man page should perhaps be updated to clarify this limitation. However, with current behaviour the rewriteOut cannot be filtered to specific clients, which seems like desirable behaviour.

**Comment by fmauchle** [comment]
could you specify wich version of radsecproxy you are using and provide a short example config of what you are trying to achieve?

Generally `rewriteIn` and `rewriteOut` configs can be applied to both clients and servers, while the In/Out always refers to the traffic direction of the client or server.
So from your description, a `rewriteOut` in a `client` block will be applied to response messages returned to that client.
